AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -283,19 +283,19 @@ - (void)testLogsNotFlushedImmediatelyWhenIntervalIsCustom {
   NSUInteger batchSizeLimit = 4;
   int itemsToAdd = 8;
   NSUInteger flushInterval = 600;
-
-  // Configure channel.
   self.sut.configuration = [[MSChannelUnitConfiguration alloc] initWithGroupId:kMSTestGroupId
                                                                       priority:MSPriorityDefault
                                                                  flushInterval:flushInterval
                                                                 batchSizeLimit:batchSizeLimit
                                                            pendingBatchesLimit:3];
 
   // When
-  for (NSUInteger i = 0; i < itemsToAdd; i++) {
-    [self.sut enqueueItem:[self getValidMockLog] flags:MSFlagsDefault];
-  }
-  [self enqueueChannelEndJobExpectation];
+  dispatch_async(self.logsDispatchQueue, ^{
```

Comment:
I don't see how it can help. Could you please explain the idea?

Reply:
Has to be investigated yet, why this actually fixes. Basically I wanted to check if its some kind of race condition (can't explain where exactly still)